components/PralineForm/index.js

```diff
@@ -0,0 +1,213 @@
+import React, { useState } from "react";
```

Comment:
```suggestion
import { useState } from "react";
```
You don't need to import `React` here because you're already using it implicitly.